lib/cdo/graphics/certificate_image.rb

```diff
@@ -179,6 +179,8 @@ def certificate_template_for(course)
       end
     elsif course == 'mee'
       'MC_Hour_Of_Code_Certificate_mee.png'
+    elsif course == ScriptConstants::OCEANS_NAME
+      'Oceans_Hour_Of_Code_Certificate.png'
```

Comment:
Can we rename this to `oceans_hoc_certificate.png`?  Pegasus usually doesn't use caps in image filenames so much.